modules/search/instant-search/components/search-result.jsx

```diff
@@ -0,0 +1,48 @@
+/** @jsx h */
+
+/**
+ * External dependencies
+ */
+import { h, Component } from 'preact';
+
+/**
+ * Internal dependencies
+ */
+import SearchResultMinimal from './search-result-minimal';
+import SearchResultProduct from './search-result-product';
+import { recordTrainTracksRender, recordTrainTracksInteract } from '../lib/tracks';
+import { RESULT_FORMAT_PRODUCT } from '../lib/constants';
+
+class SearchResult extends Component {
+	componentDidMount() {
+		recordTrainTracksRender( this.getCommonTrainTracksProps() );
+	}
+
+	getCommonTrainTracksProps() {
+		return {
+			fetch_algo: this.props.result.railcar.fetch_algo,
+			fetch_position: this.props.result.railcar.fetch_position,
+			fetch_query: this.props.result.railcar.fetch_query,
+			railcar: this.props.result.railcar.railcar,
+			rec_blog_id: this.props.result.railcar.rec_blog_id,
+			rec_post_id: this.props.result.railcar.rec_post_id,
+			ui_algo: 'jetpack-instant-search-ui/v1',
+			ui_position: this.props.index,
+		};
+	}
+
+	onClick = () => {
```

Comment:
@bluefuton I just realized we might have broken our TrainTracks integration with this change. Can you confirm?

Reply:
Good spot! Look like that's the case - a regression introduced in the refactor of `SearchResult`.

Reply:
Fixed!

Reply:
Thanks @jsnmoon! 👍